lib/interceptor.js

```diff
@@ -73,6 +73,7 @@ Interceptor.prototype.reply = function reply(statusCode, body, rawHeaders) {
 
     if (this.scope._defaultReplyHeaders) {
         headers = headers || {};
+        headers = common.headersFieldNamesToLowerCase(headers);
```

Comment:
@iangreenleaf This corrects the throwing conflict exception. Do we have test cases for testing overrides of the default headers when strings are loosely matching ('SomEthing' ==='something')?